mavros/CMakeLists.txt

```diff
@@ -130,6 +130,7 @@ add_library(mavros_plugins
   src/plugins/hil_controls.cpp
   src/plugins/hil_actuator_controls.cpp
   src/plugins/home_position.cpp
+  src/plugins/setpoint_global_position.cpp
```

Comment:
@vooon this is not a `setpoint_global_position`, as there is a specific msg for that in Mavlink. This is a setpoint for local position as we have in `setpoint_position` but with a function to convert from GPS coordinates to local frame. So it is an extension of local setpoints and not a plugin for global setpoints. It will only make sense to add a new plugin for global position setpoints if those are published in a global frame.